docs/admin/dbedit.md

```diff
@@ -0,0 +1,287 @@
+# db edit commands
+
+All db editing commands are provided in heketi binary. This is to highlight the restriction that db editing must be performed only when heketi server isn't running. If the commands are performed when the server is using the database, "Unable to open database: timeout" error is shown.
+
+IMPORTANT NOTE:
+All operations on the db must be performed when heketi server isn't operational.  
+Procudure to be followed:
+1. Shutdown the heketi service. In container platforms, do so by scale down pods to zero.
+2. mount the gluster volume "heketidbstorage" on node.
+3. Backup the heketi.db 
+4. edit the db using commands given below.
+5. umount gluster volume heketidbstorage
+6. start heketi service. In container platforms, do so by scaling up the pod to 1.
+```
+# mount -t glusterfs NODEIP:/heketidbstorage /mnt
+# cp /mnt/heketi.db /mnt/heketi.db.bak
+# heketi db <commands ...>
+# umount /mnt
+```
+
+## export command
+export creates a JSON file from a db file. Use this to check contents of heketi db without using the API. Also note, this provides complete content of the db including DB attributes bucket that heketi uses internally to keep backward compatibility and to version the database. This also includes the contents of pending operations bucket that lists operations which are ongoing or got interrupted.
```

Comment:
* check contents of heketi db -> check the contents of the heketi db
* this provides complete content -> this provides the complete content
* DB attributes bucket -> the DB attributes bucket
* pending operations bucket -> the pending operations bucket